parser-typechecker/src/Unison/Codebase/Serialization/V0.hs

```diff
@@ -539,16 +539,28 @@ putDataDeclaration :: (MonadPut m, Ord v)
                    -> DataDeclaration' v a
                    -> m ()
 putDataDeclaration putV putA decl = do
-  putA $ DataDeclaration.annotation decl
+  putModifier (DataDeclaration.modifier decl)
```

Comment:
Nothing interesting here, just needed to add serialization and deserialization of the modifier (`unique` or `structural`)